starcounter-include.html

```diff
@@ -288,6 +289,60 @@
             return range.createContextualFragment(htmlStr);
         }
 
+        /**
+         * Replaces all <slot> tags with <content>,
+         *  preserves all attributes, translates selector to slot name
+         * @param  {String} string stringified HTML
+         * @return {String}        translated HTML
+         */
+        function replaceSlotsWithContentInString(string){
+            return string.replace(/<\/slot(\s*)>/gi,'</content$1>')
```

Comment:
Well that's awesome, I checked with a two linters and with Chrome. And they both accepted it. But if W3C agrees with you, it's the linters' problem 💃